client/src/Ratings.jsx

```diff
@@ -52,6 +56,54 @@ class Ratings extends Component {
   // look into retis
   // elastic search db
   //
+
+  onAddReview(obj) {
+    console.log('In the ratings component')
```

Comment:
Not very important but console logs should not be pushed up according to the best practices module in learn